prow/tide/README.md

```diff
@@ -0,0 +1,128 @@
+# Tide
+
+One of [Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md)
+components for managing a pool of PRs that match a given set of criteria.
+Automatically retesting PRs in the pool and merging them if they pass tests.
```

Comment:
... if they pass tests against the latest base branch commit.

Reply:
Done